civihr_default_theme/templates/page/page.tpl.php

```diff
@@ -15,8 +15,18 @@
 
   $admin_link = l(t('CiviHR admin'), 'civicrm', array('html' => true));
   $ssp_link = l(t('CiviHR SSP'), 'dashboard', array('html' => true));
-  $documents_link = l(t('Manage documents'), 'admin/content', array('html' => true));
-  $users_link = l(t('Manage users'), 'admin/people', array('html' => true));
+
+  $mapGearLinks = array(
+    "administer nodes" => l(t('Manage documents'), 'admin/content', array('html' => true)),
+    "administer users" => l(t('Manage users'), 'admin/people', array('html' => true)),
+    "edit terms in 2" => l(t('HR resource types'), 'hr-resource-types-list', array('html' => true))
```

Comment:
Hello @omarabuhussein, 

Instead of "edit terms in 2", can you use the name of the vocabulary to fetch it's ID ? or is it something that won't change ?

Thanks!!